app/src/main/java/com/keylesspalace/tusky/service/AccountChooserService.kt

```diff
@@ -0,0 +1,65 @@
+/* Copyright 2019 Levi Bard
+ *
+ * This file is a part of Tusky.
+ *
+ * This program is free software; you can redistribute it and/or modify it under the terms of the
+ * GNU General Public License as published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Tusky is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even
+ * the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with Tusky; if not,
+ * see <http://www.gnu.org/licenses>. */
+
+package com.keylesspalace.tusky.service
+
+import android.annotation.TargetApi
+import android.content.ComponentName
+import android.content.IntentFilter
+import android.graphics.drawable.Icon
+import android.os.Bundle
+import android.service.chooser.ChooserTarget
+import android.service.chooser.ChooserTargetService
+import android.text.TextUtils
+import androidx.room.Room
+import com.keylesspalace.tusky.R
+import com.keylesspalace.tusky.db.AccountManager
+import com.keylesspalace.tusky.db.AppDatabase
+import com.keylesspalace.tusky.util.NotificationHelper
+import com.squareup.picasso.Picasso
+
+@TargetApi(23)
+class AccountChooserService : ChooserTargetService() {
+    lateinit var accountManager: AccountManager
+
+    override fun onCreate() {
+        super.onCreate()
+        val appDatabase = Room.databaseBuilder(applicationContext, AppDatabase::class.java, "tuskyDB")
```

Comment:
Should we pull it somewhere? Is it possible?

Reply:
Yeah, I wasn't sure about that...

Reply:
you can inject it like in `SendTootService`